data/wanderers start.txt

```diff
@@ -376,16 +376,59 @@ mission "Tell Wanderers about the Pug"
 				decline
 
 
+mission "Wanderers: Defend Vara Ke'sok Hint"
+	name "Wanderer supplies to <planet>"
+	description "Transport <cargo> to <system>, where repairs are needed from Unfettered attacks."
+	passengers 1
+	cargo "building supplies" 20
+	source 
+		near "Sich'ka'ara" 1 100
+	destination "Vara Ke'sok"
+	to offer
+		random < 60
+		has "Wanderers Conversation: offered"
+		not "Wanderers: Defend Vara Ke'sok: done"
+	on offer
+		conversation
+			`Walking around the spaceport, you are approached by two Wanderers. One of them begins speaking in the Hai language, which your translator picks up. "Greetings, [stranger, curiosity]. We heard you perform [charitable, heroic] acts. Is this correct?"`
+			choice
+				`	"Yes it is. What do you need?"`
+				`	"That sounds like me."`
+				`	"How much are you willing to pay?"`
+					goto jerk
+			`	The speaking Wanderer nods in approval.`
+				goto proposal
+			label jerk
+			`	The speaking Wanderer looks at you with a blank expression, and simply says, "There can be [compensation, arrangements] of <payment> for the work.`
+			label proposal
+			`	"At the [body, planet] named Vara Ke'sok, there are [raids, plundering] by Unfettered that take place. During these, various structures get [damaged, hurt]. We wish you to help [fix, mend] these by carrying <tons> of [supplies, materials]."`
```

Comment:
> [raids, plundering]

Plundering doesn't really fit here. Raids is a noun, but plundering is a verb. "Attacks" would be a better replacement.

Reply:
Changed plundering to conquests, hinting at the unfettered's later actions.